auth/index.js

```diff
@@ -10,7 +10,7 @@ passport.use(new Strategy((username, password, done) => {
       if (!user) {
         return done(null, false);
       }
-      if (!user.password && !bcrypt.compareSync(password, user.password)) { // eslint-disable-line no-sync
+      if (!bcrypt.compareSync(password, user.password)) { // eslint-disable-line no-sync
```

Comment:
Nothing wrong here, just want to make this visible because this doesn't really belong in this PR, but needs done. Leaving it though